ui/ui-config/pages/app/new/index.tsx

```diff
@@ -35,17 +33,19 @@ class CustomErrorHandler extends EventDispatcher<{ error: string }> {
 export default function Page(props) {
   const {
     errorHandler,
-    apps,
-    plugins,
+    availablePlugins,
+    installedPlugins,
   }: {
     errorHandler: ErrorHandler;
-    apps: Actions.AppOptions["types"];
-    plugins: Actions.PluginsAvailableList["plugins"];
+    installedPlugins: Actions.AppOptions["plugins"];
+    availablePlugins: Actions.PluginsAvailableList["plugins"];
   } = props;
```

Comment:
I think this change helps clarify what's happening on this page - we actually listing plugins that provide apps (some installed, some that could be installed).  We aren't listing apps.